File: complexity_analysis/SumForLoop.py
import time
time.time()

# timestamp1 = time.time()
timestamp1 = time.time() * 1000  # Multiply by 1000 to get milliseconds

### Python program to find the sum of n natural numbers ###
number = 100
total = 0

for value in range(1, number + 1):
    total = total + value

print("The sum is :", total)

### Program completed ###

# timestamp2 = time.time()
timestamp2 = time.time() * 1000  # Multiply by 1000 to get milliseconds

print("Execution time: ", timestamp2 - timestamp1)

File: complexity_analysis/SumForLoop.java
public class SumForLoop {

    public static void main(String[] args) {

        // Get current timestamp in milliseconds
        long timestamp1 = System.currentTimeMillis();

        // Java program to find the sum of n natural numbers
        int number = 100;
        int total = 0;

        for (int value = 1; value <= number; value++) {
            total = total + value;
        }

        System.out.println("The sum is: " + total);

        // Get current timestamp in milliseconds after program completion
        long timestamp2 = System.currentTimeMillis();
        
        System.out.println("Execution time: " + (timestamp2 - timestamp1) + " milliseconds");
    }
    
}
